package.json

```diff
@@ -1,6 +1,14 @@
 {
   "name": "@fractal-framework/fractal-contracts",
-  "version": "0.1.0",
+  "version": "1.0.0",
+  "files": [
+    "dist",
+    "contracts"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/decent-dao/fractal-contracts.git"
+  },
```

Comment:
need to add a `publish:prepare` script -- this is what it looks like in previous repo -- 
 
"publish:prepare": "npm run clean && npx rimraf dist && npm run compile && npx tsc"